src/scenes/CreateEvent/index.js

```diff
@@ -49,6 +50,7 @@ const CreateEventDialog = ({ classes, store: { createEvent: { isOpen } } }) => (
         <ResultSetStartTime />
         <ResultSetEndTime />
         <Outcomes />
+        <LanguageSelector />
```

Comment:
why a Language Selector? I thought we were going to create the event based on the currently selected language?

Reply:
ok to keep it, let's move it under the `ResultSetter` though.